spec/controllers/rents_controller_spec.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe RentsController do
+  include_context 'Authenticated User'
+
+  describe 'GET #create' do
+    context 'When creating a rent for an user' do
+      let!(:user) { create(:user) }
+      let!(:book) { create(:book) }
+      let!(:rent) { attributes_for(:rent, book_id: book.id) }
+      before do
+        post :create, params: { user_id: user.id, rent: rent }
+      end
+
+      it 'returns http success' do
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'responds with the created rent' do
+        expect(response_body).to have_key('loan')
```

Comment:
I think is not necessary to respond with the rent, since was the frontend the one who gives you the data